pages/invite/[club]/[invite]/[token]/index.js

```diff
@@ -1,7 +1,7 @@
-import renderPage from '../renderPage.js'
-import { doApiRequest, formatResponse, LOGIN_URL } from '../utils'
+import renderPage from '../../../../../renderPage.js'
+import { doApiRequest, formatResponse, LOGIN_URL } from '../../../../../utils.js'
```

Comment:
Why have `.js` on the end of these?

Reply:
Fixed, note that these changes were introduced in #59 and will also appear in that merge request.